docs/auth.js

```
@@ -45,9 +45,7 @@ class GoogleAuth {
     return GoogleAuth.getAuthInstance().signIn(oAuthOptions);
   }
   signOut() {
-    try {
-      this.getAuthInstance().signOut();
```

hah. nice silent error we had there. :)

good fix.